content/pt-br/chaos-engineering.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Engenharia do Caos
+status: Completed
+category: conceito
+tags: ["metodologia", "", ""]
+---
+
+## O que é
+
+Engenharia do Caos ou EC é a disciplina de experimentação em um [sistema distribuído](/distributed-systems/) em produção para aumentar a confiança na capacidade do sistema para resistir a condições turbulentas e inesperadas.
+
+## Problema relacionado
+
+As práticas [SRE](/pt-br/site-reliability-engineering/) e [DevOps](/pt-br/devops/) focam-se em técnicas para aumentar a resiliência e confiabilidade do produto.
+A capacidade de um sistema para tolerar falhas enquanto garante um serviço de qualidade é geralmente um requisito de desenvolvimento de software.
+Existem vários aspetos envolvidos que podem levar a interrupções da aplicação, como infraestrutura, plataforma ou outras partes móveis de uma aplicação com base em ([micro serviços](/microservices/)).
```

Comment:
here you can link to `/pt-br/microservices/` and I think to keep the translation pattern, use `Microsserviços` instead of `micro serviços`